nativescript-core/ui/bottom-navigation/bottom-navigation.android.ts

```
@@ -629,21 +635,33 @@ export class BottomNavigation extends TabNavigationBase {
             is = ImageSource.fromFileOrResourceSync(iconSource);
         }
 
-        let imageDrawable: android.graphics.drawable.BitmapDrawable;
-        if (is && is.android) {
-            let image = is.android;
+        return is && is.android;
+    }
 
+    private getDrawableInfo(image: android.graphics.Bitmap): IconInfo {
+        if (image) {
             if (this.tabStrip && this.tabStrip.isIconSizeFixed) {
                 image = this.getFixedSizeIcon(image);
             }
 
-            imageDrawable = new android.graphics.drawable.BitmapDrawable(application.android.context.getResources(), image);
+            let imageDrawable = new android.graphics.drawable.BitmapDrawable(application.android.context.getResources(), image);
+
+            return {
+                drawable: imageDrawable,
+                height: image.getHeight()
+            };
         } else {
             // TODO
             // traceMissingIcon(iconSource);
         }
 
-        return imageDrawable;
+        return null;
```

Wouldn't it be better to throw an exception here?